### PR TITLE
Feature/cli entrypoint update

### PR DESCRIPTION
**Pull request by hardbyte**
Some minor changes while reading through - closer to current best practices

### CODE REVIEW

**Comment by hardbyte** [comment]
Also I've pushed a version rebased on-top of #1 in https://github.com/hardbyte/writemore/tree/feature/cli-entrypoint-update-rebased.

File: README.md [hunk 1]
```diff
@@ -8,38 +8,38 @@ Make sure you have all the **requirements** above, if not, install/get them.
...
-Clone the repository and install the requirements (we recommend using a virtual environment like `conda`):
+Clone the repository and install the requirements (we recommend using a Python virtual environment):
 
 ```bash
-pip install -r requirements.txt
+pip install writemore
 ```
 
-Rename `.env.template` to `.env` and fill in your `OPENAI_API_KEY` from the [OpenAI dashboard](https://platform.openai.com/account/api-keys).
+The library requires the `OPENAI_API_KEY` environment variable to be set with an api key obtained from
```
Comment (giorgiop) [inline README.md]: I'd still leave some indication on how to set the `.env` for less experienced users

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "writemore"
+version = "0.1.0"
```
Comment (giorgiop) [inline pyproject.toml]:
```suggestion
version = "0.0.1"
```

**Review by giorgiop — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hardbyte) [reply README.md]: Good point - I'll add to the readme for bash/powershell.

File: README.md [hunk 3]
```diff
@@ -4,49 +4,63 @@ A library of customizable autonomous agents for common jobs.
...
+```bash
+export OPENAI_API_KEY="your_api_key_here"
+```
+
+Run the command line application python script in your terminal:
 *(Type this into your CMD window)*
 
 ```bash
-python cli.py
+python -m writemore.cli
```
Comment (ajndkr) [inline README.md:41]: we can setup a console script for CLI usage later.

**Review by ajndkr — APPROVED** [review]
lgtm!

File: README.md (hunk 3 above)
Comment (hardbyte) [reply README.md:41]: Yup I defined `writemore` as a console script - https://github.com/giorgiop/writemore/pull/2/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R9-R10

I didn't document it in the readme as I'm not sure how well that works on all platforms.

**Pull request merged**